src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.xml

```diff
@@ -164,6 +164,17 @@
             <tag name="api_platform.doctrine.orm.query_extension.collection" priority="16" />
         </service>
         <service id="ApiPlatform\Core\Bridge\Doctrine\Orm\Extension\OrderExtension" alias="api_platform.doctrine.orm.query_extension.order" />
+
+        <service id="api_platform.doctrine.orm.collection_extensions" class="ApiPlatform\Core\Bridge\Doctrine\Orm\CollectionExtensions" public="false">
```

Comment:
Why don't you use them in our providers?

Reply:
right I thought it'd be a BC break but in fact they're not so yes I will!

Reply:
Mhh in fact as it changes the signature of the DataProvider it may be a bc break

Reply:
If you change the definition to `array|\Traversable` and remove the type (`array`) in the constructor then no more BC break!